includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -358,17 +358,15 @@ public function refund_charge( $charge_id, $amount = null ) {
 	 *
 	 * @param string $intention_id - The ID of the intention to capture.
 	 * @param int    $amount       - Amount to capture.
-	 * @param array  $level3       - Level 3 data.
 	 *
 	 * @return WC_Payments_API_Intention
 	 * @throws API_Exception - Exception thrown on intention capture failure.
 	 */
-	public function capture_intention( $intention_id, $amount, $level3 = [] ) {
+	public function capture_intention( $intention_id, $amount ) {
 		$request                      = [];
 		$request['amount_to_capture'] = $amount;
-		$request['level3']            = $level3;
 
-		$response_array = $this->request_with_level3_data(
+		$response_array = $this->request(
```

Comment:
This will change the behavior of manually capturing payments from the admin area.
Instead we should supply empty level3 data when capturing in-person payments.

Reply:
With empty level3 data, won't the `request_with_level3_data` method simply pass through the request to the `request` method just like my direct call here?

https://github.com/Automattic/woocommerce-payments/blob/fa960bcd98377f53b51adbf55b854e27e1c7bc8c/includes/wc-payment-api/class-wc-payments-api-client.php#L1717-L1721

Reply:
> This will change the behavior of manually capturing payments from the admin area.

Would love to know more about it :) If I'm not wrong, the method `WC_Payments_API_Client::capture_intention` is used only in the `WC_Payment_Gateway_WCPay::capture_charge` method.

Reply:
Correct, and `WC_Payment_Gateway_WCPay::capture_charge` is also used in `add_action( 'woocommerce_order_action_capture_charge', [ $this, 'capture_charge' ] );`.

In WCPay settings we can enable authorizing payments and the from order action manually capture the payment.

Reply:
> With empty level3 data, won't the request_with_level3_data method simply pass through the request to the request method just like my direct call here?

Yes, that's the point. If we have a way from IPP capture payment endpoint pass in empty level 3 data, we are good.

Reply:
Addressed the side effect in 3a7906a